build_runner/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   build_config: ^0.3.1
   build_daemon: ^0.2.0
   build_resolvers: ^0.2.0
-  build_runner_core: ^2.0.0
+  build_runner_core: ^2.0.1
```

Comment:
You don't need to bump this - its just a bug fix. A pub upgrade will fix things for people either way.

Reply:
(no need to release a new build_runner at all)

Reply:
Not sure what I was thinking. Fixed.

Reply:
Actually I was right initially. I removed the reset in `build_runner` so the dot dep is necessary.